### ISSUE TITLE
How does one access a notebook once it's ready?

### ISSUE THREAD

**Issue by Wagk** [millejoh/emacs-ipython-notebook#480]
I've just started using this, and I can't seem to find an explanation on how to proceed after running `ein:notebook-open` on a notebook.

The minibuffer prints `%notebook% Pending open`, before printing `Worksheet is ready`, so I'm assuming it's working correctly. However, I have no idea how to proceed from here.  Pressing `c-c c-z` and `c-c c-o` just restarts the notebook opening process.

**Comment by dickmao** [comment]
`Pending open` is indicative of a lower level problem, maybe jupyter, maybe networking/filesystem level.

#481 puts self-diagnosing instructions into the README as follows:

First issue `M-x ein:dev-start-debug`. Then reproduce the error.

Higher level diagnostics appear in `M-x ein:log-pop-to-all-buffer`.

Lower level diagnostics (the actual curl requests) appear in `M-x ein:log-pop-to-request-buffer`.

If you cannot resolve the problem, file an issue using `M-x ein:dev-bug-report-template`. Please ensure the resulting system output does not include information sensitive to your institution.

**Comment by dsche-cyber** [comment]
After `M-x ein:log-pop-to-all-buffer`

In `M-x ein:log-pop-to-request-buffer` I get:
```
[debug] request--callback: executing success
[debug] request--callback: executing complete
[debug] /tmp/curl-trace unreadable
```

The debug report:
## System info:

```cl
("EIN system info"
 :emacs-version
 "GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.22, cairo version 1.17.3)
 of 2020-08-28"
 :emacs-bzr-version nil
 :window-system x
 :emacs-variant nil
 :os
 (:uname
  "Linux dsche-pc 5.8.11-1-MANJARO #1 SMP PREEMPT Wed Sep 23 14:35:40 UTC 2020 x86_64 GNU/Linux
"
  :lsb-release
  "LSB Version:	n/a
Distributor ID:	ManjaroLinux
Description:	Manjaro Linux
Release:	20.1.1
Codename:	Mikah
")
 :notebook
 "Name: notebook
Version: 6.1.4
Summary: A web-based notebook environment for interactive computing
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: /usr/lib/python3.8/site-packages
Requires: jinja2, tornado, pyzmq, argon2-cffi, ipython-genutils, traitlets, jupyter-core, jupyter-client, nbformat, nbconvert, ipykernel, Send2Trash, terminado, prometheus-client
Required-by: widgetsnbextension, jupyterlab, jupyterlab-server
"
 :ipython "7.18.1
"
 :image-types
 (svg png gif tiff jpeg xpm xbm pbm)
 :image-types-available
 (svg png gif tiff jpeg xpm xbm pbm)
 :request
 (:backend curl)
 :ein
 (:version nil
           :source-dir "/home/dsche/.emacs.d/.local/straight/build/ein/")
 :lib
 ((:name "websocket"
         :path "~/.emacs.d/.local/straight/build/websocket/websocket.elc"
         :featurep t
         :version-var websocket-version
         :version "1.12")
  (:name "anaphora"
         :path "~/.emacs.d/.local/straight/build/anaphora/anaphora.elc"
         :featurep t
         :version-var nil
         :version nil)
  (:name "request"
         :path "~/.emacs.d/.local/straight/build/request/request.elc"
         :featurep t
         :version-var request-version
         :version "0.3.0")
  (:name "deferred"
         :path "~/.emacs.d/.local/straight/build/deferred/deferred.elc"
         :featurep t
         :version-var deferred:version
         :version "0.5.0")
  (:name "polymode"
         :path "~/.emacs.d/.local/straight/build/polymode/polymode.elc"
         :featurep t
         :version-var nil
         :version nil)
  (:name "exec-path-from-shell"
         :path
         "~/.emacs.d/.local/straight/build/exec-path-from-shell/exec-path-from-shell.elc"
         :featurep t
         :version-var nil
         :version nil)
  (:name "dash"
         :path "~/.emacs.d/.local/straight/build/dash/dash.elc"
         :featurep t
         :version-var nil
         :version nil)
  (:name "with-editor"
         :path "~/.emacs.d/.local/straight/build/with-editor/with-editor.elc"
         :featurep t
         :version-var nil
         :version nil)))
```
